File: windows/ClipboardMaster.Core/HistoryLimits.cs
namespace ClipboardMaster.Core;

/// <summary>全局常量集中管理（与 macOS 版保持一致）。</summary>
public static class HistoryLimits
{
    /// <summary>历史保留条数上限。</summary>
    public const int Capacity = 200;

    /// <summary>单条文本存储字符上限（边界校验）。</summary>
    public const int StoredTextCap = 100_000;

    /// <summary>菜单条目标题预览长度。</summary>
    public const int MenuPreviewLimit = 60;

    /// <summary>悬浮提示预览长度。</summary>
    public const int TooltipPreviewLimit = 200;
}

File: Sources/ClipboardMasterCore/HistoryLimits.swift
import Foundation

/// 全局常量集中管理，避免硬编码散落各处。
public enum HistoryLimits {
    /// 历史保留条数上限。
    public static let capacity = 200
    /// 剪贴板轮询间隔（秒）。
    public static let pollInterval: TimeInterval = 0.5
    /// 单条文本存储字符上限（边界校验）。
    public static let storedTextCap = 100_000
    /// 菜单条目标题预览长度。
    public static let menuPreviewLimit = 60
    /// 悬浮提示预览长度。
    public static let tooltipPreviewLimit = 200
}
